packages/apps/ds4ch/components/Blog/BlogAuthor.vue

```diff
@@ -0,0 +1,25 @@
+<script setup>
+defineProps({
+  name: {
+    type: String,
+    required: true,
+  },
+  organisation: {
+    type: String,
+    default: "",
+  },
+  url: {
+    type: String,
+    default: "",
+  },
+});
+</script>
+<template>
+  <address>
```

Comment:
I'm not sure that [address](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/address) is really appropriate here.

Reply:
Okay then.